k8s/webpackager/README.md

```diff
@@ -0,0 +1,308 @@
+# Overview
+
+Web Packager HTTP Server is an HTTP server built on top of Web Packager.
+It functions like a reverse-proxy, fetching documents from a backend server,
+then optimizing and signing them before returning them to requestors. This
+is similar to [AMP Packager][], but Web Packager targets all HTML documents
+except for AMP documents, whereas AMP Packager packages AMP documents. It aims
+to meet the [requirements][] set by the Google SXG Cache.
+
+Currently, if you need to package [AMP][] documents into a signed exchange,
+it is recommended that you use [AMP Packager][] for that purpose and use
+[Web Packager][] for everything else. This may change in the future where only
+one packager does both jobs, but for now it means that you have to set up both
+packagers if you need to process both AMP and non-AMP content.
+
+[AMP]: https://amp.dev/
+[AMP Packager]: https://github.com/ampproject/amppackager
+[requirements]: https://github.com/google/webpackager/blob/master/docs/cache_requirements.md
+[Web Packager]: https://github.com/google/webpackager
+
+To learn more about Web Packager, see the [Web Packager Github Site](https://github.com/google/webpackager).
+
+## About Google Click to Deploy
+
+Popular open stacks on Kubernetes packaged by Google.
+
+## Architecture
+
+![Architecture diagram](resources/webpackager-k8s-app-architecture.png)
+
+This application uses 2 replicas of the Web Packager with a load balancer
+service in front that routes requests to the instances. The number of replicas
+is configurable.  All Web Packager Pods are associated with an NFS Server as 
+their PersistentVolume which is created as a standard persistent disk defined by
+Google Kubernetes Engine.
+
+This application exposes one endpoint: HTTPS on (configurable) port 6000.
+
+This application requires a Signed Exchange Certificate in order to function.
+The application, through its ACME Configuration, automatically requests and
+retrieves this certificate from the Certificate Authority. The application also
+periodically (approximately every 90 days) requests automatic renewal of the
+certificate.
+
+If you want to use this application in a production environment, you must:
+
+*   Configure it to use your [DigiCert ACME account][] to obtain your
+certificate.
+
+[DigiCert ACME Account]:https://docs.digicert.com/manage-certificates/certificate-profile-options/get-your-signed-http-exchange-certificate/.
+
+# Installation
+
+## Quick install with Google Cloud Marketplace
+
+Get up and running with a few clicks! Install this Web Packager app to a Google
+Kubernetes Engine cluster using Google Cloud Marketplace. Follow the
+[on-screen instructions](https://console.cloud.google.com/marketplace/details/google/webpackager).
+
+## Command line instructions
```

Comment:
I think I mentioned this amppackager and you said "we can't" but if there's any way we can either move this section below the usage/scaling instructions, or extract to a different doc and link to it, I think that would better meet the needs of most readers.

Reply:
Yeah the README for the deployers follow a certain format.